tests/verifyQAN_test.js

```diff
@@ -46,4 +46,52 @@ Scenario('Open the QAN Dashboard and verify pagination', async (I, adminPage, qa
     qanPage.checkTableHeaders();
     //qanPage.checkServerList();
     // I.switchTo();
+});
+
+Scenario('Open the QAN Dashboard and verify Tables tab in Query Details for Database=postgres filter', async (I, adminPage, qanPage) => {
+    I.amOnPage(qanPage.url);
+    await I.waitForElement(qanPage.fields.iframe, 60);
+    adminPage.applyTimer("5m");
+    await I.switchTo(qanPage.fields.iframe); // switch to first iframe
+    qanPage.waitForQANPageLoaded();
+    qanPage.applyFilter("postgres");
+    qanPage._selectDetails(2);
+    qanPage.selectSectionInDetails(qanPage.fields.tablesTabInDetails);
+    await qanPage.verifyDetailsSectionDataExists();
+});
+
+Scenario('Open the QAN Dashboard and verify Tables tab in Query Details for Environment=pgsql-dev filter', async (I, adminPage, qanPage) => {
+    I.amOnPage(qanPage.url);
+    await I.waitForElement(qanPage.fields.iframe, 60);
+    adminPage.applyTimer("5m");
+    await I.switchTo(qanPage.fields.iframe); // switch to first iframe
+    qanPage.waitForQANPageLoaded();
+    qanPage.applyFilter("pgsql-dev");
+    qanPage._selectDetails(2);
+    qanPage.selectSectionInDetails(qanPage.fields.tablesTabInDetails);
+    await qanPage.verifyDetailsSectionDataExists();
+});
+
+Scenario('Open the QAN Dashboard and verify Explain tab in Query Details for Database=postgres filter', async (I, adminPage, qanPage) => {
+    I.amOnPage(qanPage.url);
+    await I.waitForElement(qanPage.fields.iframe, 60);
+    adminPage.applyTimer("5m");
+    await I.switchTo(qanPage.fields.iframe); // switch to first iframe
+    qanPage.waitForQANPageLoaded();
+    qanPage.applyFilter("postgres");
+    qanPage._selectDetails(2);
+    qanPage.selectSectionInDetails(qanPage.fields.explainTabInDetails);
+    await qanPage.verifyExplainDetailsSectionDataExists();
+});
+
+Scenario('Open the QAN Dashboard and verify Explain tab in Query Details for Environment=pgsql-dev filter', async (I, adminPage, qanPage) => {
+    I.amOnPage(qanPage.url);
+    await I.waitForElement(qanPage.fields.iframe, 60);
+    adminPage.applyTimer("5m");
+    await I.switchTo(qanPage.fields.iframe); // switch to first iframe
```

Comment:
when you switch to Iframe, when do you switch back?

Reply:
i do not need it